webapp-django/crashstats/settings/base.py

```diff
@@ -721,6 +721,11 @@ def filter(self, record):
 # Number of seconds before we retry a failed job
 CRONTABBER_ERROR_RETRY_TIME = 300
 
+# Configuration for the depcheck command
+NPM_PATH = config('NPM_PATH', '/usr/bin/npm')
+SAFETY_PATH = config('SAFETY_PATH', '/usr/local/bin/safety')
+SAFETY_API_KEY = config('SAFETY_API_KEY', '')
+PACKAGE_JSON_PATH = config('PACKAGE_JSON_PATH', '/app/webapp-django/package.json')
```

Comment:
Three of these have path values to binaries in the docker image, so I just set them. The API key will be set in the ops code. We'll need to set that when this lands.